tests/test_docker.py

```diff
@@ -733,31 +733,47 @@ def pre(req):
     assert end - start > 1
 
 
+def assert_ping_stat_resources(resp):
+    hostname = Config.hostname()
+    pool_name = hostname + ' Storage Pool'
+    assert resp['data']['resources'][0]['name'] == hostname
+    assert resp['data']['resources'][1]['name'] == pool_name
+    resp['data']['resources'][0]['name'] = 'localhost'
+    resp['data']['resources'][1]['name'] = 'localhost Storage Pool'
+
+
 def ping_post_process(req, resp):
-        hostname = Config.hostname()
-        pool_name = hostname + ' Storage Pool'
-        resources = resp['data']['resources']
+    resources = resp['data']['resources']
 
-        uuid = 'c861f990-4472-4fa1-960f-65171b544c28'
-        instances = filter(lambda x: x['type'] == 'instance' and
-                           x['uuid'] == uuid, resources)
-        assert len(instances) == 1
-        instance = instances[0]
-        assert instance['dockerId'] is not None
-        del instance['dockerId']
+    uuids = ['uuid-running', 'uuid-stopped', 'uuid-created']
```

Comment:
Craig, don't see uuid-created being set anywhere in this PR. I assume its in the code outside of this PR?